demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -130,6 +135,7 @@ def rotate_poses(poses_3d, R, t):
         cv2.putText(frame, 'FPS: {}'.format(int(1 / mean_time * 10) / 10),
                     (40, 80), cv2.FONT_HERSHEY_COMPLEX, 1, (0, 0, 255))
         if args.no_show:
+            frame = cap.read()
```

Comment:
I suggest to move the part working with windows under `if not args.no_show` instead, so you won't need to have `frame = cap.read()` twice.

Reply:
Done